duckduckgo/duckduckgo-help-pages
# Copy Updates

- Various copy updates
- Create new Settings/Params page

Demo: https://moollaza-duckduckgo-duckduckgo-help-pages-979p9j4497cp4w9-4000.githubpreview.dev/

## Testing Instructions 

- Click "Code" top-right, and create a codespace 
- In the codespace Terminal...
  - run `bundle install`
  - run `bundle exec jekyll serve --baseurl ""`
- Click the "Ports" tab
- Click the globe icon beside port 4000 (this should open the dev server in a new tab)
- [ ] Confirm the new params page exists in the "Settings" category
- [ ] Confirm the partnerships page is updated
